src/endpoints/v1.es6.js

```diff
@@ -182,6 +183,27 @@ class APIv1Endpoint {
     cache.set(key, data);
   }
 
+  get subreddits () {
+    return bind({
+      buildOptions: function(options) {
+        var uri = this.origin + `/r/${options.query.subreddit}/about.json`;
```

Comment:
:no_bell: we may need to move uri building into the actual methods (`get`, `post`) instead of in buildOptions at some point. Not a change you need to make - but this is going to get messy as we start to add more options to query subreddits.